src/components/UsercentricsProvider.tsx

```diff
@@ -28,27 +25,19 @@ interface UsercentricsProviderProps {
     windowEventName: string
 }
 
-/**
- * The provider component for listening to events from the Usercentrics Browser API.
- * Render this once and wrap your application in it.
- *
- * @example <caption>Basic usage</caption>
- * () => <UsercentricsProvider windowEventName="ucEvent"><App /></UsercentricsProvider>
- *
- * @example <caption>Custom timeout in milliseconds</caption>
- * () => <UsercentricsProvider timeout={100} windowEventName="ucEvent"><App /></UsercentricsProvider>
- */
 export const UsercentricsProvider: FC<UsercentricsProviderProps> = ({
     children,
     strictMode = false,
-    timeout = 5000,
+    timeout = 15000,
     windowEventName,
 }) => {
+    const [state, setState] = useState<ServiceState>(UserCentericsService.state)
+    UserCentericsService.subscribe(setState)
```

Comment:
This calls `UserCentericsService.subscribe()` on every render of the provider. Should we move it to a `useEffect` instead? I'm pretty sure we can leave the deps array empty, so it will only run once.

Reply:
I wonder if it would be technically cleaner to also supply `UserCentericsService.unsubscribe(<some-id>)`... 🤔 Maybe the use-case doesn't really require it.

Reply:
the use-effect is a good point, ill make it